display_ui.rb

```diff
@@ -0,0 +1,94 @@
+require_relative 'app'
+
+class DisplayUI
+  def initialize(app)
+    @app = app
+  end
+
+  def list_all_books
+    puts 'Books:-'
+    @app.list_all_books
+  end
+
+  def list_all_peoples
+    puts 'Peoples:-'
+    @app.list_all_peoples
+  end
+
+  def create_book
+    puts 'Title: '
+    title = gets.chomp
+    puts 'Author: '
+    author = gets.chomp
+    @app.create_book(title, author)
+  end
```

Comment:
- [ ] I can see a lot of input requests in your methods that create, I will like to remind you of YAGNI and KISS which you learned [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid), it would be better to have all requests for input and input validation placed in the main.rb file so that it would satisfy the principles earlier pointed out.

Reply:
Hi @Olamarx ,
Thanks for review.
Added input and input validation in main.rb